libs/gltfio/include/gltfio/FilamentAsset.h

```diff
@@ -194,14 +194,13 @@ class UTILS_PUBLIC FilamentAsset {
     const char* getExtras(utils::Entity entity = {}) const noexcept;
 
     /**
-     * Lazily creates the animation engine or returns it from the cache.
+     * Returns the animation engine.
      *
      * The animator is owned by the asset and should not be manually deleted.
-     * The first time this is called, it must be called before FilamentAsset::releaseSourceData().
      * If the asset is instanced, this returns a "primary" animator that controls all instances.
      * To animate each instance individually, use \see FilamentInstance.
      */
-    Animator* getAnimator() noexcept;
+    Animator* getAnimator() const noexcept;
```

Comment:
Can this only be called after `ResourceLoader::loadResources`?

Reply:
Yes this is still true, good catch, I will update the docs.

Reply:
`Animator` should have a private destructor (to prevent the user calling delete on it), it should also have deleted copy/move ctors and copy operators (or private if you need that internally).

Reply:
It already has a private destructor, I'll go ahead and delete copy/move methods.

Reply:
Done.